rules/xcodeproj.bzl

```diff
@@ -323,6 +326,19 @@ def _xcodeproj_impl(ctx):
     xcodeproj_schemes_by_name = {}
 
     for target_info in targets:
+        target_name = target_info.name
+
+        if "xcodeproj-ignore-as-target" in target_info.tags:
+            continue
+
+        if target_name in xcodeproj_targets_by_name:
+            existing_type = xcodeproj_targets_by_name[target_name]["type"]
+            if target_info.product_type != existing_type:
+                fail("""
+Target "{}" is defined with type "{}", but a same-name target of type "{}" wants to override.
```

Comment:
can we get the full labels of the conflicting targets in the error message somehow?

Reply:
added a line before this to indicate which label we are in

Reply:
but what about the labels of the iOS targets that the error is saying are conflicting?